ansible/playbook/tasks/update.yml

```diff
@@ -25,9 +25,26 @@
       when: ansible_virtualization_type != "docker"
       copy:
         src: supervisord.service
-        dest: /etc/systemd/system/
+        dest: /usr/lib/systemd/system/supervisord.service
         mode: 0644
 
+    - name: Check if old supervisord service confiuration exists
+      when: ansible_virtualization_type != "docker"
+      stat: path=/etc/systemd/system/supervisord.service checksum_algorithm=md5
+      register: is_old_supervisord
+
+    - name: Check and write in log if old supervisord service confiuration was modified.
+      when: ansible_virtualization_type != "docker" and is_old_supervisord.stat.exists and is_old_supervisord.stat.checksum != "9c618e7860d822053193faca9240ed9f"
+      fail:
+        msg: '/etc/systemd/system/supervisord.service was modified - please ensure it is compatible with updates in /etc/systemd/system/supervisord.service'
+      ignore_errors: yes
+
+    - name: Remove old supervisord service confiuration
```

Comment:
We need to back up it and only after that remove . And add this info to changelog  because customer should know that this file will be removed and where he can find backup (if this file contains custom info).

Reply:
> We need to back up it and only after that remove

Why? We don't expect users to even know about that file.

Reply:
I was recently reminded that according to our policy we do not support changes made by a user.